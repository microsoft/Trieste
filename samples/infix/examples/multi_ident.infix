x = 5 + 10;
y = 1 - 9 + x;
print "1" x + y;
z = (5 * x) / y;
print "2" z;