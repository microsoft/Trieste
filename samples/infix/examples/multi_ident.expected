//! calculate
//! --enable-tuples calculate
//! --enable-tuples --tuples-require-parens calculate
//! --enable-tuples --tuples-require-parens --use-parser-tuples calculate
1 <- 22
2 <- 10
